### PR TITLE
Feat removed sodium

### PR DESCRIPTION
**Pull request by mfmurray**
Android updates
- Improved Google one tap experience

iOS Updates
- Automations
- Removed Sodium package

### CODE REVIEW

File: example/ios/Podfile.lock [hunk 1]
```diff
@@ -393,7 +393,7 @@ PODS:
     - ReSwiftThunk/Core (= 2.0.1)
   - ReSwiftThunk/Core (2.0.1):
     - ReSwift (~> 6.0)
-  - Rownd (2.8.3):
+  - Rownd (2.9.0):
```
Comment (BobbyRadford) [inline example/ios/Podfile.lock:396]: Were we going to do a major version bump?

Comment (mfmurray) [reply example/ios/Podfile.lock:396]: I think we wanted to do a major version bump because of the sodium package removal

Comment (BobbyRadford) [reply example/ios/Podfile.lock:396]: Major bump would be 3.0.0

Comment (mfmurray) [reply example/ios/Podfile.lock]: Updated

**Review by BobbyRadford — APPROVED** [review]

**Pull request merged**